### ISSUE TITLE
[Feature Request] Adding torrent clients to Dashboard

### ISSUE THREAD

**Issue by needo37** [Hellowlol/HTPC-Manager#11]
Would it be difficult to add the torrent clients to the dashboard?

**Comment by Hellowlol** [comment]
What do you think of? List a list of torrents or all features like stop/add and so on? I think I'll add a a client status where you see download upload speed of the different clients.

**Comment by needo37** [comment]
I think it would be cool to have a list of torrents currently being downloaded and maybe just a count of the torrents that are seeding. I like the download/upload speed idea!

**Comment by Hellowlol** [comment]
I wont be coding this feature myself, if any wants to chip it ill happily merge it. :)

**Comment by needo37** [comment]
I'd be happy to help.

**Comment by Hellowlol** [comment]
Nice! Im doing some work on the dash now. I want the position of the "modules" to be drag and droppable. But i cant seem save the damn order or disable the soring.

**Comment by Hellowlol** [comment]
@needo37 There was a pr for qbittorrent on the dash for some time ago. Would something like that be enough? What torrent clients are you using?

**Comment by fma965** [comment]
I know this is a old ticket but i wanted to get deluge on the dashboard, tried to figure it out but since deluge webinterfaces can connect to multiple daemons it becomes more confusing than it first seemed.

**Comment by Hellowlol** [comment]
Isnt it very unlikely that its more then one daemon?

**Comment by fma965** [comment]
Yes but i guess what i meant was that the deluge code which isn't for the dashboard currently has a option to select daemon before it connects (unless it's recently been connected) and this is a bit harder to do on the dashboard so it would need a lot different code from what is used in deluge.js, hope that makes sense, sorry for the replies on various tickets i'm just bored and want to contribute where i can :)

**Comment by Hellowlol** [comment]
Nothing to be sorry about. Im happy anybody takes interest. Well what about just auto connect to the first available?

**Comment by fma965** [comment]
Yeah that could work, i'm not sure... i would like to get this working as it's obvious relevant to my own usage.
